coap/message.go

```diff
@@ -0,0 +1,184 @@
+package adapter
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net"
+
+	"github.com/dereulenspiegel/coap-mux"
+	"github.com/dustin/go-coap"
+	"github.com/mainflux/mainflux/coap/nats"
+	"github.com/mainflux/mainflux/normalizer"
+	broker "github.com/nats-io/go-nats"
+)
+
+type (
+	Observer struct {
+		Conn    *net.UDPConn
+		Addr    *net.UDPAddr
+		Message *coap.Message
+	}
+)
+
+// Map of observers
+var ObsMap map[string][]Observer
+
+// sendMessage function
```

Comment:
This is not exported function, and its name is quite clear without repeating it in the comment.